src/FilterMonitor.js

```diff
@@ -10,10 +10,11 @@ export default class FilterMonitor extends Component {
     blacklist: PropTypes.array
   };
 
-  isFiltered(action) {
+  isFiltered(actionType) {
+    var type = actionType || '';
```

Comment:
I mean that if there's no action type we don't have to check it as `''` will not be find in the filtered actions'  array.

Anyway, having an undefined action type is antipatern and Redux should through before coming here. Could you please try to update Redux DevTools as [this pr](https://github.com/gaearon/redux-devtools/pull/232) should forbid having such action types?